index.html

```diff
@@ -118,7 +118,7 @@ <h1>
       <div class="left-content">
```

Comment:
- [ ] I am just using this place to request a change about `linter checks` which are not related to this index.html. The project requirements say the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) and [HTML&CSS](https://github.com/microverseinc/linters-config/tree/master/html-css) linters should be set correctly for this project. Currently, your repo does not have eslint check. Kindly fix it 🙏

Reply:
And already fixed that div! thank you